contributing.md

```diff
@@ -5,7 +5,7 @@ By participating in this project you agree to abide by its terms.
 
 ## Add quality
 
-Ensure your pull request adheres to the following guidelines
+Ensure your pull request adheres to the following guidelines.
```

Comment:
Following the sentence, there is an enumeration. Please use a `:`.

Reply:
sorry i clicked resolve by mistake im changing it in sometime. thank you for your response.